### ISSUE TITLE
Odd interaction with Factorization Water Boilers

### ISSUE THREAD

**Issue by esotericist** [Vswe/ModJam3#30]
While the A29 update handled the problems I was having with Factorization inventories not realizing they have materials, I'm now having trouble with Water Boilers.

I'm able to put water into them with SFM without any trouble, and they do seem to create steam (tested with a fluiduct on top of the boiler), SFM seems to be unable to pull steam from the boiler.

**Comment by Vswe** [comment]
It could be that the factorization boilers aren't giving out their tank info properly.

**Comment by SomewhatDamaged** [comment]
To pull steam out of the boiler you must blacklist water from being extracted. (tested)

You can pull from the firebox or the boiler tank itself.

**Comment by Vswe** [comment]
That's the wrong type of boiler.

**Comment by esotericist** [comment]
SomewhatDamaged: We're talking about factorization water boilers (a single block), not railcraft boilers (a multiblock structure).
And yes, I tried both whitelist (steam) and blacklist (water). Right now, the best theory is as vswe said; the fz boilers aren't exposing their tank information properly.

**Comment by SomewhatDamaged** [comment]
Yeah, sorry. Retested on the Factorization ones and they do indeed seem not to work. I apologize for my overzealousness.
